owners/index.js

```diff
@@ -14,17 +14,19 @@
  * limitations under the License.
  */
 
-const sleep = require('sleep-promise');
+const Octokit = require('@octokit/rest');
 const {GitHub, PullRequest} = require('./src/github');
 const {LocalRepository} = require('./src/local_repo');
-const {CheckRun, OwnersCheck} = require('./src/owners_check');
+const {OwnersBot} = require('./src/owners_bot');
+const {OwnersParser} = require('./src/owners');
+const {OwnersCheck} = require('./src/owners_check');
 
-const GITHUB_CHECKRUN_DELAY = 2000;
+const GITHUB_TOKEN = process.env.GITHUB_TOKEN;
 
 module.exports = app => {
-  app.on(['pull_request.opened', 'pull_request.synchronize'], onPullRequest);
-  app.on('check_run.rerequested', onCheckRunRerequest);
-  app.on('pull_request_review.submitted', onPullRequestReview);
+  const localRepo = new LocalRepository(process.env.GITHUB_REPO_DIR);
```

Comment:
Does this also pull the repo? If it does, that means the repo will be synced only once, on app startup... (if this doesn't do the pulling, ignore)

Reply:
Ignoring :upside_down_face:

Reply:
There's a separate `LocalRepository.checkout` method. I thought doing file system ops in a constructor might be a bad idea :laughing: